.travis.yml

```diff
@@ -5,11 +5,17 @@ python:
   - 3.3
   - 3.4
   - 3.5
-  - pypy
-  - pypy3
+  - 3.6
+  - pypy-5.4.1
```

Comment:
pypy3.3 won't ever get beyond alpha status AFAIU (plus general python 3.3 support ends soon too). The pypy3 devs are focusing all of their attention on pypy3.5, which is currently in beta. It's possible to test that on travis, but it takes some work.

Given the presence of a pypy2 and a python 3.3, and a python 3.5, I think we have the possible combos covered pretty well, with the exception of alpha- or beta-specific bugs, and given how fast pypy dev is moving now, it's probably not really worth adding workarounds for those, at least in my opinion.